### PR TITLE
Install kustomize with "go get"

### PR DESCRIPTION
**Pull request by fmuyassarov**

### CODE REVIEW

**Comment by nordixinfra** [comment]
Can one of the admins verify this patch?

**Comment by maelk** [comment]
add to whitelist

File: resources/docker/workspace/Dockerfile [hunk 1]
```diff
@@ -43,8 +43,7 @@ RUN curl -LO https://raw.githubusercontent.com/golang/dep/master/install.sh && \
     chmod +x ./kubectl && \
     mv ./kubectl /usr/local/bin/kubectl && \
     \
-    curl -LO https://raw.githubusercontent.com/Nordix/cluster-api-provider-baremetal/master/tools/install_kustomize.sh && \
-    chmod +x ./install_kustomize.sh && ./install_kustomize.sh && rm install_kustomize.sh && \
+    @if ! which kustomize >/dev/null 2>&1 ; then echo "Running 'go get sigs.k8s.io/kustomize'" && go get sigs.k8s.io/kustomize ; fi \
     mv /data/go/bin/kustomize /usr/local/bin && \
```
Comment (maelk) [inline resources/docker/workspace/Dockerfile]: the mv command should be part of the if

**Comment by maelk** [comment]
it looks like you have a second commit there that is part of another PR. Should it not be there ?

**Pull request closed without merging**